lib/tweetStream.js

```diff
@@ -0,0 +1,30 @@
+const Twitter = require('twitter')
+const {
+  CONSUMER_KEY,
+  CONSUMER_SECRET,
+  ACCESS_TOKEN_KEY,
+  ACCESS_TOKEN_SECRET
+} = process.env
```

Comment:
Can we have a check that all the env variables are set correctly and throw an error ? because the error thrown by the twitter library doesn't really make sense